src/main/java/seedu/address/model/Hotel.java

```diff
@@ -115,6 +118,19 @@ public boolean hasBooking(Booking booking) {
         return false;
     }
 
+    /**
+     * Checks if {@code person} has booked {@code room}.
+     */
+    public boolean hasGuestBooked(Person person, Room room) {
+        for (Booking b : bookingList) {
```

Comment:
I'm pretty sure bookingList contains future reservation as well. Have you check for this case? I think we can create a method to check whether it's a reservation, or a stay. Clarify with me if this is not clear.